staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -35,7 +35,9 @@ var (
 	// MetricNameLabel is label under which model.Sample stores metric name
 	MetricNameLabel model.LabelName = model.MetricNameLabel
 	// QuantileLabel is label under which model.Sample stores latency quantile value
-	QuantileLabel model.LabelName = model.QuantileLabel
+	QuantileLabel                             model.LabelName = model.QuantileLabel
+	frameworkExtensionPointDurationMetricName                 = "scheduler_framework_extension_point_duration_seconds"
+	ExtensionPointLabelName                                   = "extension_point"
```

Comment:
I'm +1 on @ahg-g comment. We shouldn't define these in the generic lib.

Reply:
Done